### PR TITLE
[15.0][MIG]project_task_material: Migration v15

### PR DESCRIPTION
**Pull request by bizzappdev**

### CODE REVIEW

File: project_task_material/__manifest__.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 {
     "name": "Project Task Material",
     "summary": "Record products spent in a Task",
-    "version": "14.0.1.0.0",
+    "version": "15.0.0.0.1",
```
Comment (AaronHForgeFlow) [inline project_task_material/__manifest__.py]: `"version": "15.0.1.0.0",`

**Comment by bizzappdev** [comment]
> Hey @bizzappdev, thank you for your Pull Request.
> 
> It looks like some users haven't signed our **C**ontributor **L**icense **A**greement, yet. You can read and sign our full Contributor License Agreement here: http://odoo-community.org/page/cla Here is a list of the users:
> 
> * DHR [DHR@bizzappdev.com](mailto:DHR@bizzappdev.com) (no github login found)
> 
> Appreciation of efforts, OCA CLAbot

Changes for the author is done.

**Review by rafaelbn — APPROVED** [review]
👍

**Comment by rafaelbn** [comment]
@dreispt @AaronHForgeFlow please 😄 2 minutes!

**Review by dreispt — APPROVED** [review]

**Pull request merged**